components/Form/DatePicker/DatePicker.tsx

```diff
@@ -1,9 +1,18 @@
 import { TextInput } from '..';
+import { TextInputNoType } from 'components/Form/types';
+import { format } from 'date-fns';
 
-import type { TextInputNoType as Props } from 'components/Form/types';
+interface Props extends TextInputNoType {
+  defaultToday?: boolean;
+}
 
-const DateInput = (props: Props): React.ReactElement => (
-  <TextInput width={10} {...props} type="date" />
+const DateInput = ({ defaultToday, ...props }: Props): React.ReactElement => (
+  <TextInput
+    width={10}
+    {...props}
+    type="date"
+    defaultValue={defaultToday ? format(new Date(), 'yyyy-MM-dd') : undefined}
```

Comment:
This is more of a question, with the format yyyy-MM-dd I get the display would be 2021-05-27 for today, does the date information still contain the time and could that also be changed.

I know we're trying to target the ISO-8061 standard more often for consistent datetime so am keen to not trim some of the time data.

Reply:
this is only a `date` picker. there's a seperate `time` element that we might have use for in future.

it's a text input with extra trimmings. the underlying text format is always yyyy-mm-dd, but the user's browser and operating system settings control exactly how it renders.

so in short, it holds no time information at all